mws/data_mws_credentials.go

```diff
@@ -9,6 +9,7 @@ import (
 
 func DataSourceMwsCredentials() common.Resource {
 	type mwsCredentialsData struct {
+		common.Namespace
```

Comment:
This shouldn't be here because this should only be used for workspace-level resources.